persistence/testkit/src/main/scala/com/lightbend/lagom/internal/persistence/testkit/PersistenceTestConfig.scala

```diff
@@ -11,52 +11,58 @@ import com.typesafe.config.ConfigFactory
 private[lagom] object PersistenceTestConfig {
 
   lazy val BasicConfigMap: Map[String, AnyRef] = Map(
-    "lagom.akka.management.enabled" -> "off"
+    "lagom.akka.management.enabled" -> "off",
+    "akka.actor.provider"           -> "local",
```

Comment:
we should disable remote in basic config and override it when enabling clustering